src/App.jsx

```diff
@@ -1,40 +1,32 @@
-import React from 'react';
+// import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { useState } from 'react';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${
+          tabs.find(tab => tab.id === selectedTabId).title
```

Comment:
```suggestion
          tabs.find(tab => tab.id === selectedTabId)?.title
```
find can return the undefined